portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+
+public final class Task {
```

Comment:
Please change to use 2 spaces for indentation instead of 4 (which also matches other files like DataServlet.java)

https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation